stl/inc/istream

```diff
@@ -48,13 +48,13 @@ public:
     }
 
 protected:
-    __CLR_OR_THIS_CALL basic_istream(basic_istream&& _Right) : _Chcount(_Right._Chcount) {
+    __CLR_OR_THIS_CALL basic_istream(basic_istream&& _Right) noexcept : _Chcount(_Right._Chcount) {

```

Comment:
It seems unsuitable to add `noexcept` (ditto for `basic_iostream` and `basic_ostream`), since
- this constructor calls `basic_ios<...>::init`,
- `init` calls `widen`,
- `widen` calls `use_facet`, which may throw `bad_cast`.

However, it seems OK to make `move` functions `noexcept`.